records_mover/db/mysql/load_options.py

```diff
@@ -1,9 +1,8 @@
 from sqlalchemy.sql.expression import text, TextClause
-from records_mover.records import RecordsHints
 from records_mover.utils import quiet_remove
 from records_mover.records.hints import cant_handle_hint
 from records_mover.records import DelimitedRecordsFormat
-from records_mover.records.types import (
+from records_mover.records.hints import (
     HintEncoding, HintRecordTerminator,
     HintFieldDelimiter, HintQuoteChar,
     HintQuoting, HintEscape,
```

Comment:
I refactored a bunch of things into a new module called hints.  I'm pondering a rename to 'delimited', with possible a future re-parenting of 'format' (so maybe someday this would be something like `from records_mover.records.format.delimited`)